docs/installing.md

```diff
@@ -9,19 +9,58 @@ Kubernetes version.
 
 ## macOS
 
+### Requirements
+
 Rancher Desktop requires the following on macOS:
 
-- macOS 10.10 or higher.
+- macOS Catalina 10.15 or higher.
 - Intel CPU with VT-x.
 - Persistent internet connection.
 
 Apple Silicon (M1) support is planned, but not currently implemented.
 
+It is also recommended to have:
+
+- 8 GB of memory
+- 4 CPU
+
+Additional resources may be required depending on the workloads you plan to run.
+
+### Installing Rancher Desktop on macOS
+
+1. Go to the [releases page] on Github.
+1. Find the version of Rancher Desktop you want to download.
+1. Expand the **Assets** section and download either `Rancher.Desktop-X.Y.Z-mac.zip` or `Rancher.Desktop-X.Y.Z.dmg`, where `X.Y.Z` is the version of Rancher Desktop.
```

Comment:
We should just always point people at the `.dmg` for installation. The zip file is mostly used for updates.  (Also, it's technically not an installer — that would be a `.pkg`.)

Reply:
Removed part about zip.